nipype/algorithms/misc.py

```diff
@@ -245,7 +245,7 @@ def _run_interface(self, runtime):
         else:
             affine = None
 
-        with open(self.inputs.header_file, 'rb') as data_file:
+        with open(self.inputs.data_file, 'rb') as data_file:
```

Comment:
Thanks @effigies. @satra, my workflows were working until I got this update, it only worked when I fixed this. A Nipype Camino tractography example also uses this, it probably won't work without this. I was thinking on adding a test for this case, but I would need Analyze files, somehow. Any hint?